source/plot/SvgPlot.ooc

```diff
@@ -104,7 +105,7 @@ SvgPlot: class {
 		}
 	}
 	fillColorList: func {
-		this colorList = VectorList<String> new()
+		this colorList = VectorList<String> new(10, false)
```

Comment:
Then we need a better color representation but that can be a separate pull request since the string mess already existed.